jf_agent/main.py

```diff
@@ -370,15 +370,18 @@ def download_data(config, creds, endpoint_jira_info, endpoint_git_instances_info
             print_all_jira_fields(config, jira_connection)
         download_data_status.append(load_and_dump_jira(config, endpoint_jira_info, jira_connection))
 
-    
+    if len(config.git_configs) == 0: 
+        return download_data_status
```

Comment:
`ThreadPoolExecutor(max_workers=0)` throws a ValueError so I decided to short-circuit here